verify/verify.go

```diff
@@ -98,6 +106,18 @@ func VerifyToken(c *messagebird.Client, id, token string) (*Verify, error) {
 	return verify, nil
 }
 
+func GetVerifyEmailMessage(c *messagebird.Client, id string) (*VerifyMessage, error) {
```

Comment:
I'd suggest renaming to `ReadVerifyEmailMessage` for consistency with other methods for fetching serverside resources

Reply:
done